pkg/detectors/stormboard/stormboard.go

```diff
@@ -0,0 +1,74 @@
+package stormboard
+
+import (
+	"context"
+	"fmt"
```

Comment:
It won't compile with this unused import. Also, the test seems to be failing right now.

Reply:
Oh the test failing might be my very slow wifi timing out. Please remove the unused import though.